src/components/Book.js

```diff
@@ -1,45 +1,76 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { useState } from 'react';
 import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
 import { useDispatch } from 'react-redux';
-import styles from '../Styles/Book.css';
-import { removeBook, getBooksFromApi } from '../redux/books/booksSlice';
+import styles from '../styles/Book.module.css';
+import { getBooksFromApi, displayNewBook } from '../redux/books/booksSlice';
```

Comment:
- [ ] When i tried to run the project locally, I observed that the styles are not being loaded properly. The issue lies in the incorrect import path for the stylesheets. Currently, they have been imported as `../styles/Book.module.css`, but it seems the actual folder name starts with a capital `S`, making it `../Styles/Book.module.css`. This is a case-sensitivity problem. To resolve it and ensure the stylesheets are loaded correctly, please update the import paths to match the exact folder name, `Styles,` with a capital `S`."